superset/models/core.py

```diff
@@ -320,7 +324,7 @@ def get_quoter(self):
         return self.get_dialect().identifier_preparer.quote
 
     def get_df(  # pylint: disable=too-many-locals
-        self, sql: str, schema: str, mutator: Optional[Callable] = None
+        self, sql: str, schema: Optional[str] = None, mutator: Optional[Callable] = None
```

Comment:
`self.get_sla_engine` sets schema to None by default so I moved this up a level to make my db query easier (since it's not schema dependent)